webapp-django/crashstats/crashstats/views.py

```diff
@@ -871,15 +871,7 @@ def report_index(request, crash_id, default_context=None):
         context['raw_stackwalker_output'] = 'No dump available'
         parsed_dump = {}
 
-    # If the parsed_dump lacks a `parsed_dump.crash_info.crashing_thread`
```

Comment:
I traced this back to https://github.com/mozilla-services/socorro/pull/2237. It seems like it was added to avoid an UndefinedError on an older version of the report index, but the current version (with the modifications made in this PR) handle a missing `crashing_thread` value without a problem.

Reply:
👍